tasks/bootstrap_local_vm/02_create_local_vm.yml

```diff
@@ -115,7 +115,7 @@
         changed_when: true
       - name: Create local VM
         command: >-
-          virt-install -n {{ he_vm_name }}Local --os-variant rhel7 --virt-type kvm --memory {{ he_mem_size_MB }}
+          virt-install -n {{ he_vm_name }}Local --os-variant rhel8.0 --virt-type kvm --memory {{ he_mem_size_MB }}
```

Comment:
Can it be variable instead of hard-coded?

Reply:
currently, we support only 1 variant of the appliance .